ui/src/router/components/form/components/autoscaling_policy/AutoscalingPolicyPanel.js

```diff
@@ -93,8 +93,10 @@ export const AutoscalingPolicyPanel = ({
               onChange={(e) => onChangeTarget(e.target.value)}
               isInvalid={!!errors.target}
               name="memory"
-              min={1}
-              step={1}
+              // The min value is set as 0.005 because it's the smallest value, when rounded to 2 decimal places, gives
+              // 0.01, the smallest value accepted as an autoscaling target (concurrency).
+              min={0.005}
+              step={"any"}
```

Comment:
The setting the step size to `any` (the default value is `1`) is to fix this issue whereby a red line appears when a value that is outside the configured step size (e.g. 1.003 when the step size is 1) is set:
![Screenshot 2024-05-09 at 11 56 32 AM](https://github.com/caraml-dev/turing/assets/36802364/a330e25f-ec2d-4b7a-b892-8e2fb938c419)

The min value has also been changed,  to allow values below 0 (e.g. 0.5) to be set, which are valid values when the concurrency autoscaling target is used.